src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -855,6 +850,9 @@ void TwoPointFluxApproximation::computeBoundaryStencil( MeshLevel & mesh,
   arrayView2d< localIndex const > const & elemList           = faceManager.elementList();
   arrayView2d< real64 const, nodes::REFERENCE_POSITION_USD > const & nodePosition = nodeManager.referencePosition();
 
+  arrayView1d< real64 const > const & transMultiplier =
+    faceManager.getReference< array1d< real64 > >( m_coeffName + viewKeyStruct::transMultiplierString() );
```

Comment:
At some point, we should also remove the `m_coeffName` member, but this can be done later (after the thermal work is merged maybe). 

If I remember well, this class data member is only used to register the `transMultiplier` and attach it to the physical coefficient it will multiply. But this  prevents us from easily reusing the existing `computeWeights` function if we ever want to apply the multiplier differently on different physics (not sure there is a good use case, but we never know). So I would vote for removing the `m_coeffName` member, moving the multiplier to the solver, and making the transmissibility multiplier an argument of `computeWeights`.

Reply:
Yes, we should do this. The only problem is that to access the multiplier from the physics solver kernel we need the original connector (e.g. face) index, which is currently not stored with the stencil, but it won't be a problem to add it.